spec/lib/granite/rspec/raise_validation_error_spec.rb

```diff
@@ -43,13 +43,13 @@ def execute_perform!(*)
     specify do
       expect do
         expect { action.perform! }.to raise_validation_error.on_attribute(:raise_error)
-      end.to fail_with('expected to raise validation error on attribute :raise_error, but raised {:base=>[{:error=>:some_error}], :raise_error=>[]}')
+      end.to fail_with(include 'expected to raise validation error on attribute :raise_error, but raised {:base=>[{:error=>:some_error}]')
```

Comment:
The extra reported error was an artifact due to older activemodel initializing a message array on access (and our usage of `deep_merge!`). Test is changed to only check the relevant part of the message.